packages/e2e-tests/specs/experiments/widgets.test.js

```diff
@@ -0,0 +1,363 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	deactivatePlugin,
+	activatePlugin,
+	activateTheme,
+} from '@wordpress/e2e-test-utils';
+
+/** @typedef {import('puppeteer').ElementHandle} ElementHandle */
+
+describe( 'Widgets screen', () => {
+	beforeEach( async () => {
+		await visitAdminPage( 'themes.php', 'page=gutenberg-widgets' );
+		// Wait for the widget areas to load.
+		await page.waitForSelector(
+			'[aria-label="Block: Widget Area"][role="group"]'
+		);
+	} );
+
+	afterEach( async () => {
+		await cleanupWidgets();
+	} );
+
+	beforeAll( async () => {
+		// TODO: Ideally we can bundle our test theme directly in the repo.
+		await activateTheme( 'twentytwenty' );
+		await deactivatePlugin(
+			'gutenberg-test-plugin-disables-the-css-animations'
+		);
+		await cleanupWidgets();
+	} );
+
+	afterAll( async () => {
+		await activatePlugin(
+			'gutenberg-test-plugin-disables-the-css-animations'
+		);
+		await activateTheme( 'twentytwentyone' );
+	} );
+
+	async function getParagraphBlockInGlobalInserter() {
```

Comment:
Nit: Why are some of these helper functions at the top here and some at the bottom of the file? 😀

Reply:
It's just a pattern more aligned to my mental model 😆 . Functions here are regular utility functions, but functions at the bottom should eventually be moved to the `e2e-test-utils` package once they're stable.